header_test.go

```diff
@@ -129,16 +127,54 @@ func TestRequestRawHeaders(t *testing.T) {
 		if string(h.Host()) != "foobar" {
 			t.Fatalf("unexpected host: %q. Expecting %q", h.Host(), "foobar")
 		}
-		v1 := h.Peek("EmptyValue")
-		if len(v1) > 0 {
-			t.Fatalf("expecting empty value. Got %q", v1)
```

Comment:
Why the need to remove EmptyValue testing?

Reply:
EmptyValue check is not the purpose if this Test. It has been copied alongside other checks (by me, in #27) from a general headers tests that remains unchanged.

In other words EmptyValue is tested elsewhere, the removal is my attempt to DRY my previous PR.